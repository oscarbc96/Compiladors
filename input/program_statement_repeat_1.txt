calc off

a := 5

repeat 
  123 
  123 
  123
  a := a + 1
until (a < 7)
