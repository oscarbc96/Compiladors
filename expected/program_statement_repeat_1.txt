1: $t01 := 5
2: a := $t01
3: PARAM 123
4: CALL PUTI,1
5: PARAM 123
6: CALL PUTI,1
7: PARAM 123
8: CALL PUTI,1
9: $t02 := a ADDI 1
10: a := $t02
11: IF a LTI 7 GOTO 3
12: GOTO 13
13: HALT
